.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -396,15 +396,18 @@ enum class TestType(val unitTests: Boolean = true, val functionalTests: Boolean
     forceRealizeDependencyManagement(false, true, false)
 }
 
-enum class PerformanceTestType(val taskId: String, val timeout: Int, val defaultBaselines: String = "", val extraParameters: String = "") {
+enum class PerformanceTestType(val taskId: String, val timeout: Int, val defaultBaselines: String = "", val extraParameters: String = "", val uuid: String? = null) {
     test("PerformanceTest", 420, "defaults"),
-    experiment("PerformanceExperiment", 420, "defaults"),
+    // TODO: Rename to `slow`
+    experiment("SlowPerformanceTest", 420, "defaults", uuid = "PerformanceExperimentCoordinator"),
     flakinessDetection("FlakinessDetection", 420, "flakiness-detection-commit"),
     historical("FullPerformanceTest", 2280, "2.14.1,3.5.1,4.0,last", "--checks none");
 
-    fun asId(model: CIBuildModel): String {
-        return "${model.projectPrefix}Performance${name.capitalize()}Coordinator"
-    }
+    fun asId(model: CIBuildModel): String =
+        "${model.projectPrefix}Performance${name.capitalize()}Coordinator"
+
+    fun asUuid(model: CIBuildModel): String =
+        uuid?.let { model.projectPrefix + it } ?: asId(model)
```

Comment:
I would prefer `uuid:String = ""` and 

```
if(uuid.isEmpty()) asId(model) else "${model.projectPrefix}$uuid"
```

Reply:
I prefer using null, so I don't need to keep in mind that the empty string needs special handling.